gum/arch-arm/gumarmreader.c

```diff
@@ -125,3 +128,10 @@ disassemble_instruction_at (gconstpointer address)
 
   return insn;
 }
+
+static guint
+gum_rotate_right_32bit (guint val, guint rotation)
```

Comment:
These should be split across multiple lines like elsewhere in frida-gum (but not like in frida-core where we follow a different convention).